tools/lint/config/eslint.config.mjs

```diff
@@ -0,0 +1,229 @@
+import { fixupConfigRules } from '@eslint/compat';
+import * as emotion from '@emotion/eslint-plugin';
+import importPlugin from 'eslint-plugin-import';
+import simpleImportSort from 'eslint-plugin-simple-import-sort';
+import globals from 'globals';
+import babelParser from '@babel/eslint-parser';
+import typescriptEslint from '@typescript-eslint/eslint-plugin';
+import tsParser from '@typescript-eslint/parser';
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+import js from '@eslint/js';
+import { FlatCompat } from '@eslint/eslintrc';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const compat = new FlatCompat({
+  baseDirectory: __dirname,
+  recommendedConfig: js.configs.recommended,
+  allConfig: js.configs.all,
+});
+
+export default [
+  ...fixupConfigRules(
+    compat.extends(
+      'eslint:recommended',
+      'plugin:react/recommended',
+      'plugin:react-hooks/recommended',
+      'plugin:jest/recommended',
+      'plugin:jsx-a11y/recommended',
+      'plugin:storybook/recommended',
+    ),
+  ),
```

Comment:
Was this the fix to get ignores to work, or something else?

Reply:
no this just part of their migration script

Reply:
What were we doing incorrectly?